recipe/meta.yaml

```diff
@@ -1,39 +1,45 @@
-{% set version = "0.4.6" %}
+{% set name = "WSGIProxy2" %}
+{% set version = "0.5.1" %}
 
 package:
-  name: wsgiproxy2
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/W/WSGIProxy2/WSGIProxy2-{{ version }}.tar.gz
-  sha256: cee2a64abc193cc96c7ff1208b709335e9a4d1316ce84b91501102166d814c9a
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0d9ecb141de720e2fd4f7a275a4a83a961ffeb6717483d940021ffa1c46f665c
 
 build:
   number: 0
-  noarch: python
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation  -vv
 
 requirements:
   host:
-    - python
     - pip
+    - python
+    - setuptools >=61.0.0
   run:
     - python
-    - setuptools
     - webob
     - six
```

Comment:
https://github.com/gawel/WSGIProxy2/blob/3194f1a208e0022c0efa97824fd740661e0a99f2/CHANGES.rst?plain=1#L13
`six` is no longer required